app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor["name"]} should be vaccinated")
```

Comment:
The string formatting here is incorrect. You should use single quotes around the key `name` inside the f-string to avoid syntax errors: `f"{visitor['name']} should be vaccinated"`.